ffs/api/api.go

```diff
@@ -120,15 +183,15 @@ func (i *API) GetCidConfig(c cid.Cid) (ffs.CidConfig, error) {
 	return conf, nil
 }
 
-// SetDefaultCidConfig sets a new default CidConfig.
-func (i *API) SetDefaultCidConfig(c ffs.DefaultCidConfig) error {
+// SetDefaultConfig sets a new default CidConfig.
+func (i *API) SetDefaultConfig(c ffs.DefaultConfig) error {
```

Comment:
Renamed things like "DefaultCidConfig" just simply "DefaultConfig" because in the default case, there is nothing about a cid in there and it was confusing since there is also `GetDefaultCidConfig` which prepares a `CidConfig` from the default using a cid. This makes it easier to understand.